docs/releases/5.0.4.txt

```diff
@@ -9,4 +9,5 @@ Django 5.0.4 fixes several bugs in 5.0.3.
 Bugfixes
 ========
 
-* ...
+* Fixed a bug in Django 5.0 where validation of ``UniqueConstraint`` using
+  ``OpClass`` would crash with an SQL syntax error (:ticket:`35275`).
```

Comment:
This patch doesn't qualify for a backport so release notes are not needed, please remove them.